cmd/spanner-aip-go/main.go

```diff
@@ -19,7 +19,7 @@ const generatedBy = "spanner-aip-go"
 
 func main() {
 	log.SetFlags(0)
-	configFilePath := flag.String("config", "spanner-aip-go.yaml", "config file")
+	configFilePath := flag.String("config", "spanner.yaml", "config file")
```

Comment:
this is a breaking change, so commit should be prefixed with `feat!:`.

I like `spanner-aip-go.yaml` better because it better reflects what it is used for.